### PR TITLE
[cloud-provider-aws] Getting rid of unused associatePublicIPToNodes arg

### PR DESCRIPTION
**Pull request by apolovov**
## Description
Getting rid of unused `associatePublicIPToNodes` and `associatePublicIPToMasters` args.

## Why do we need it, and what problem does it solve?
It confuses people with Standard layout installed.

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: candi
type: fix
summary: Got rid of unused `associatePublicIPToNodes` and `associatePublicIPToMasters` args in AWS Standard layout.
impact_level: low
```

### CODE REVIEW

File: candi/cloud-providers/aws/openapi/cluster_configuration.yaml [hunk 1]
```diff
@@ -205,8 +205,6 @@ apiVersions:
         properties:
           associatePublicIPToMasters:
             type: boolean
```
Comment (RomanenkoDenys) [inline candi/cloud-providers/aws/openapi/cluster_configuration.yaml:207]:
```suggestion
          associatePublicIPToMasters:
            type: boolean
```
I think we remove this code too

File: candi/cloud-providers/aws/openapi/cluster_configuration.yaml [hunk 2]
```diff
@@ -203,10 +203,6 @@ apiVersions:
         required: []
         description: Layout is **deprecated**.
         properties:
```
Comment (apolovov) [inline candi/cloud-providers/aws/openapi/cluster_configuration.yaml:205]:
```suggestion
        properties: {}
```

**Review by RomanenkoDenys — COMMENTED** [review]
Matrix tests failed. May be rebase needed ?

**Review by RomanenkoDenys — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'e2e/run/aws'...

**Review by z9r5 — APPROVED** [review]

**Review by RomanenkoDenys — APPROVED** [review]
LGTM

**Pull request merged**